mods/ra/maps/in-the-nick-of-time/in-the-nick-of-time.lua

```diff
@@ -0,0 +1,412 @@
+--[[
+   Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+StartUnits = { "jeep", "jeep", "1tnk", "1tnk", "1tnk", "1tnk", "2tnk", "2tnk" }
+StartingPlanes = { Yak1, Yak2 }
+ReinforcementFootprint1 = { CPos.New(93,78), CPos.New(94,78), CPos.New(95,78) }
+InfantrySquad = { "e1", "e1", "e1", "e3", "e3" }
+ReinforcementFootprint2 = { CPos.New(63,84), CPos.New(63,85), CPos.New(63,86), CPos.New(63,87) }
+SouthWaterPath1 = { SouthWaterEntry.Location, SouthWaterLanding1.Location }
+SouthWaterPath2 = { SouthWaterEntry.Location, SouthWaterLanding2.Location }
+SouthWaterTeam = { "dtrk", "1tnk", "1tnk", "2tnk", "2tnk" }
+SouthChinookPath = { ChinookEntry1.Location, ChinookLZ1.Location }
+SouthChinookChalk = { "e3", "e3", "e3", "e1", "e1" }
```

Comment:
This took me a few attempts to get the reinforcements and was super disappointing.  I think it's better to just let the nuke truck blow it up.

Reply:
> The spy seems a bit pointless. I managed to disguise my spy and then immediately ran him onto a mine!

Every reinforcement is true to the original.